### DISCUSSION TITLE
I want Evaluation Metrics like Precision, Recall, Average Inference Time and Confusion matrix

### DISCUSSION THREAD

**Discussion by prasaddev97** [ultralytics/yolov5#6408] [Q&A] [answered]
Can we evaluate Precision, Recall, Average Inference Time and Confusion matrix using this repository ?

**Comment by glenn-jocher** [comment, marked as answer]
@prasaddev97 YOLOv5 metrics include, Precision, Recall, mAP@0.5, mAP@0.5:0.95, for all classes and also per class.

Confusion matrices and profiling results are also automatically produced by val.py.

**Reply by prasaddev97** [reply]
Thanks Sir for quick response
